Generator/Commands/ContainerGenerator.php

```diff
@@ -211,19 +287,23 @@ public function getUserInputs()
                 '--ui'          => $ui,
             ]);
 
-            Artisan::call('apiato:generate:action', [
-                '--container'   => $containerName,
-                '--file'        => $route['action'],
-                '--model'       => $model,
-                '--stub'        => $route['stub'],
-            ]);
+            if ($route['action'] != null || $route['stub'] != null) {
```

Comment:
see above.. don't check on the `null` values but rather on the `$ui`.. would be cleaner, i think!?

Reply:
Could you provide an example how to use `$ui` to check if specific route required a Task/Action?

Reply:
Why not use a `hasAction => true/false` flag?

However, i am more fond of a new `container:web` generator that is specifically used for creating a web container?! Because this would highly reduce the complexity.. what do you think?